src/modules/menu/components/header/HeaderWithSearch.vue

```diff
@@ -48,8 +49,10 @@ function updateHeaderHeight() {
 }
 
 function resetApp() {
-    // an app reset means we keep the lang and the current topic but everything else is thrown away
-    window.location = `${window.location.origin}?lang=${currentLang.value}&topic=${currentTopicId.value}`
+    // an app reset means we keep the lang, the current topic, and its default background layer but everything else is thrown away
+    // We keep the default background layer of the current topic because the app always set to `ech` topic and its default background layer before we can even get the topic from the URL
+    const defaultBackgroundLayerId = currentTopic.value?.defaultBackgroundLayer?.id
```

Comment:
I'd maybe guard against a `null` value here by having something like
```
const defaultBackgroundLayerId = currentTopic.value?.defaultBackgroundLayer?.id ?? currentBackground.value
```
What do you think?

Reply:
yes, I think it's better to have a guard. I was thinking that all topic has a default background layer. Added new.